loopd/swapclient_server.go

```diff
@@ -563,8 +565,11 @@ func (s *swapClientServer) ListSwaps(ctx context.Context,
 	req *looprpc.ListSwapsRequest) (*looprpc.ListSwapsResponse, error) {
 
 	var (
-		rpcSwaps = []*looprpc.SwapStatus{}
-		idx      = 0
+		rpcSwaps      = []*looprpc.SwapStatus{}
+		rpcSwapInfo   = []*loop.SwapInfo{}
```

Comment:
I propose to rename `rpcSwapInfo` to `swapInfos`. It is not a proto object (hence no "rpc") and it is a list (hence "s" in the end).